packages/editor/src/components/document-bar/index.js

```diff
@@ -98,12 +100,11 @@ export default function DocumentBar() {
 	const { open: openCommandCenter } = useDispatch( commandsStore );
 	const isReducedMotion = useReducedMotion();
 
-	const isNotFound = ! document && ! isResolving;
+	const isNotFound = ! documentTitle && ! isResolving;
```

Comment:
Isn't it possible to have a document but no title?

Reply:
Good catch! Should be fixed now.